extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,46 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live stories
+
+If the `live-story` attribute is specified on the `<amp-story>` element, it will allow users to receive real-time updates coming from the server. It updates the client device without having to refresh the page and shows a notification when there are updates available and the user is on the last page of the story.
+
+<figure class="centered-fig">
+  <amp-anim alt="Live stories example" width="300" height="533" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/live-stories-gif.gif">
+  <noscript>
+    <img alt="Live stories example" width="200" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/live-stories-gif.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+**Note**: As long as the `live-story` attribute is present on the `<amp-story>` element, the client will make continous polls to the server. Make sure to set the `live-story-disabled` attribute to make the polling stop once you're finished with the live broadcast of your story.
+
+Usage:
+* Import the `amp-live-list` extension in the `<head>` tag of your document.
```

Comment:
Wait, can't we dynamically load the script?

Reply:
(only if the `live-story` attribute is present)

Reply:
Is this enforced by a validation rule?

Reply:
Being addressed by #23074